engine/apps/api/serializers/direct_paging.py

```diff
@@ -44,21 +46,24 @@ class DirectPagingSerializer(serializers.Serializer):
     title = serializers.CharField(required=False, default=None)
     message = serializers.CharField(required=False, default=None, allow_null=True)
     source_url = serializers.URLField(required=False, default=None, allow_null=True)
-    grafana_incident_id = serializers.CharField(required=False, default=None, allow_null=True)
 
     def validate(self, attrs):
         organization = self.context["organization"]
         alert_group_id = attrs["alert_group_id"]
         title = attrs["title"]
         message = attrs["message"]
         source_url = attrs["source_url"]
-        grafana_incident_id = attrs["grafana_incident_id"]
+        grafana_incident_id = self.ALLOWS_GRAFANA_INCIDENT_ID and attrs.get("grafana_incident_id")
 
         if alert_group_id and (title or message or source_url or grafana_incident_id):
             raise serializers.ValidationError(
-                "alert_group_id and (title, message, source_url, grafana_incident_id) are mutually exclusive"
+                f"alert_group_id and (title, message, source_url{', grafana_incident_id' if self.ALLOWS_GRAFANA_INCIDENT_ID else ''}) "
+                "are mutually exclusive"
```

Comment:
this is here because the `grafana_incident_id` is really only a concept that is relevant for the internal API (you cannot specify this attribute from the public API)